### PR TITLE
Migrate to Flotta

### PR DESCRIPTION
**Pull request by jakub-dzon**
This PR migrates the k4e-operator code to use `Flotta` name.

### CODE REVIEW

File: docs/design/design.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Agent is running constantly on the edge device and responsible for:
 
 Agent is running in two processes:
  - [yggdrasil](https://github.com/jakub-dzon/yggdrasil/) - gateway service responsible for communication with the control plane; dispatches messages to and from the control plane from/to workers (extensions);
- - [device-worker](https://github.com/jakub-dzon/k4e-device-worker) - yggdrasil worker responsible for performing all the logic listed above.
+ - [device-worker](https://github.com/flotta/flotta-device-worker) - yggdrasil worker responsible for performing all the logic listed above.
```
Comment (gciavarrini) [inline docs/design/design.md:46]: I know that device-worker is still not present, but I think this should be `https://github.com/project-flotta/flotta-device-worker`

File: docs/user-guide/running.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ On the build machine:
   - gpgme-devel
   - device-mapper-devel
   - podman
-- Checkout https://github.com/jakub-dzon/k4e-device-worker repository 
+- Checkout https://github.com/jakub-dzon/flotta-device-worker repository
```
Comment (gciavarrini) [inline docs/user-guide/running.md:85]: Ditto.

File: docs/design/design.md (hunk 1 above)
Comment (jakub-dzon) [reply docs/design/design.md:46]: Yes! Thank you!

**Review by eloycoto — APPROVED** [review]

**Review by gciavarrini — APPROVED** [review]

**Review by bardielle — APPROVED** [review]

**Review by masayag — COMMENTED** [review]
/lgtm

**Pull request merged**